src/app/Main/Header/Export/ExportServices/DataExportServices/generateDataRowsCsvString.service.ts

```diff
@@ -12,17 +13,24 @@ import { modifyTableDataStringService } from "../../../../../Core/modifyTableDat
  */
 export function generateDataRowsCsvString(
     dataArray: DbCollection,
-    headers: DbKey[]
+    headers: DbKey[],
 ): string {
     const csvTable: string[] = [];
     dataArray.forEach((row) => {
         const csvRow: string[] = [];
+
+        const resistanceRowArray = new Array(resistancesCollection.length).fill(0)
+
         headers.forEach((element) => {
-            if (element !== "resistance") {
-                const rowValue: string = row[element];
-                csvRow.push(modifyTableDataStringService(rowValue));
+            if (element === "resistance") {
+                row.resistance.forEach((resistance) => {
+                    const resistanceIndex = resistancesCollection.indexOf(resistance) 
+                    resistanceRowArray[resistanceIndex] = 1
+                })
+                const rowValue = resistanceRowArray.join(",");
+                csvRow.push(rowValue);

```

Comment:
It would be more consistent to concat the resistanceRowArray to the csvRow array, because the csvRow is comma separated at the end anyway: ``csvRow = csvRow.concat(resitanceRowArray)``

Reply:
Done.